.github/workflows/check-tests.yml

```diff
@@ -56,8 +56,16 @@ jobs:
             */*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
 
-      - name: Test 🧪
+      - name: Prepare Tests
         working-directory: './packages/millicast-sdk'
         run: |
-          npm install
-          npm test
+          echo -n ${{secrets.MILLICAST_TEST_ACCOUNT}} | base64 -d > ./tests/test-env.js
```

Comment:
We don't need use any .env file in Github Workflow.  Instead, we could use env variables in job, such as the `NODE_AUTH_TOKEN` in `new-release` workflow.
Check this: [Defining environment variables for a single workflow](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow)